src/handlers/poster-hubble/components/index.scss

```diff
@@ -0,0 +1,18 @@
+* {
+  -webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
+  -moz-box-sizing: border-box; /* Firefox, other Gecko */
+  box-sizing: border-box; /* Opera/IE 8+ */
+}
+
+body {
+  transition: background ease-in-out;
+  font-family: 'Raleway', sans-serif;
+  //font-size: 1.75em;
```

Comment:
?

Reply:
Yeah this was already in the original index.scss file copied over, I can remove it, but not sure if the other needs to be removed or not.